templates/mock.html

```diff
@@ -0,0 +1,31 @@
+<div id="mock-section" class="section-profile">
+    <div class="row">
+         <div class="topleft duk-icon"><img onclick="removeSection('mock-section')" src="/gui/img/x.png"></div>
+         <div class="column" style="flex:40%;padding:15px;text-align: left">
+            <h1 style="font-size:70px;margin-top:-20px;">Mock</h1>
+            <h2 style="margin-top:-55px">simulated agents</h2>
+            <h5>Mock adds a set of simulated agents to CALDERA and allows you to run complete operations without hooking any other computers up to your server. All simulated agents will appear as normal agents in the Chain plugin GUI and can be used in any operation.</h5>
```

Comment:
Nit-picky bit about accessibility, but I think this should be a <p> tag instead of <h5>. Headings on a webpage should go in numerical order (so this would be h3), but the content is not a heading so I think a <p> tag would be more appropriate here.

Reply:
Good catch, changed that to a `p` tag (also in the SSL MR I made too)